modules/dnn/src/layers/cpu_kernels/conv_winograd_f63.cpp

```diff
@@ -51,6 +50,23 @@ int runWinograd63(InputArray _input, InputArray _fusedAddMat, OutputArray _outpu
     int pad_left = conv->pad_left;
 
     int ngroups = conv->ngroups, Cg = C/ngroups, Kg = K/ngroups;
+
+    static const int CONV_WINO_KBLOCK = 4;
+#if (CV_NEON && CV_NEON_AARCH64)
+    static const int CONV_WINO_IBLOCK = 6;
+#elif  CV_TRY_AVX || CV_TRY_AVX2
+    static const int CONV_WINO_IBLOCK = (conv->useAVX || conv->useAVX2) ? 6 : 3;
+#else
+    static const int CONV_WINO_IBLOCK = 3;
+#endif
+
+#if CV_TRY_AVX || CV_TRY_AVX2
+    static const int CONV_WINO_ATOM_F32 = (conv->useAVX || conv->useAVX2) ? 8 : 4;
+#else
+    static const int CONV_WINO_ATOM_F32 = 4;
+#endif
+    static const int CONV_WINO_NATOMS_F32 = CONV_WINO_AREA / CONV_WINO_ATOM_F32; // for AVX2, it is 8, otherwise, it's 16.
```

Comment:
it makes sense to use static, if they are class fields. In case if they are local variables static does not make sense. I propose just remove static from locals.